x-pack/plugins/fleet/server/integration_tests/cloud_preconfiguration.test.ts

```diff
@@ -312,31 +312,11 @@ describe('Fleet preconfiguration reset', () => {
                   cluster: ['cluster:monitor/main'],
                   indices: [
                     {
-                      names: ['logs-apm.app-default'],
+                      names: ['traces-*', 'logs-*', 'metrics-*'],
                       privileges: ['auto_configure', 'create_doc'],
                     },
                     {
-                      names: ['metrics-apm.app.*-default'],
-                      privileges: ['auto_configure', 'create_doc'],
-                    },
-                    {
-                      names: ['logs-apm.error-default'],
-                      privileges: ['auto_configure', 'create_doc'],
-                    },
-                    {
-                      names: ['metrics-apm.internal-default'],
-                      privileges: ['auto_configure', 'create_doc'],
-                    },
-                    {
-                      names: ['metrics-apm.profiling-default'],
-                      privileges: ['auto_configure', 'create_doc'],
-                    },
-                    {
-                      names: ['traces-apm.rum-default'],
-                      privileges: ['auto_configure', 'create_doc'],
-                    },
-                    {
-                      names: ['traces-apm.sampled-default'],
+                      names: ['traces-apm.sampled-*'],
```

Comment:
@axw Does these changes look good to you?

Reply:
perfect, thank you!